frontend/components/forms/fields/SelectTargetsDropdown/SelectTargetsInput/SelectTargetsInput.jsx

```diff
@@ -80,7 +84,7 @@ class SelectTargetsInput extends Component {
         resetValue={[]}
         scrollMenuIntoView={false}
         tabSelectsValue={false}
-        value={selectedTargets}
+        value={uniqueSelectedTargets}
         valueKey="unique_key" // must be unique, target ids are not unique
```

Comment:
Once we're generating uuids, this key can be called uuid. The reason is that uuids are a specific kind of unique id, so we want to indicate that in the key name.